qa/workunits/rbd/rbd_mirror_ha.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/sh
```

Comment:
@dillaman The tests starts/stops rbd-daemons, while under teuthology it is out of the script control.

I suppose I can modify the script so when it is being run under teuthology it executes only a subset of tests (may be using "leader release" instead of stop in some cases). I will investigate this tomorrow. May be you already have some hints or suggestions?